src/containers/oracle.js

```diff
@@ -221,6 +223,9 @@ TopicPage.propTypes = {
   match: PropTypes.object,
   onBet: PropTypes.func,
   betResult: PropTypes.object,
+  walletAddrs: PropTypes.array,
+  walletAddrsIndex: PropTypes.number,
+
 };
 
 TopicPage.defaultProps = {
```

Comment:
oh this class should prob be renamed to `OraclePage` or `OracleDetails`. Forgot to change that sorry.